src/warehouses/components/WarehouseSettings/WarehouseSettings.tsx

```diff
@@ -0,0 +1,188 @@
+  const intl = useIntl();
+
+  const isPrivateChoices = [
+    {
+      label: (
+        <>
+          <FormattedMessage defaultMessage="Private Stock" />
+          <Typography variant="caption" color="textSecondary">
+            <FormattedMessage defaultMessage="If enabled stock in this warehouse won't be shown" />
+          </Typography>
+          <FormSpacer />
+        </>
+      ),
+      value: "true"
+    },
+    {
+      label: (
+        <>
+          <FormattedMessage defaultMessage="Public Stock" />
+          <Typography variant="caption" color="textSecondary">
+            <FormattedMessage defaultMessage="If enabled stock in this warehouse will be shown" />
+          </Typography>
+        </>
+      ),
+      value: "false"
+    }
+  ];
+
+  const clickAndCollectChoicesPublic = [
+    {
+      label: (
+        <>
+          <FormattedMessage defaultMessage="Disabled" />
+          <Typography variant="caption" color="textSecondary">
+            <FormattedMessage defaultMessage="If selected customer won't be able to choose this warehouse as pickup point" />
+          </Typography>
+          <FormSpacer />
+        </>
+      ),
+      value: WarehouseClickAndCollectOptionEnum.DISABLED
+    },
+    {
+      label: (
+        <>
+          <FormattedMessage defaultMessage="Local stock only" />
+          <Typography variant="caption" color="textSecondary">
+            <FormattedMessage defaultMessage="If selected customer will be able to choose this warehouse as pickup point. Ordered products will be only fulfilled from this warehouse stock" />
+          </Typography>
+          <FormSpacer />
+        </>
+      ),
+      value: WarehouseClickAndCollectOptionEnum.LOCAL
+    },
+    {
+      label: (
+        <>
+          <FormattedMessage defaultMessage="All warehouses" />
+          <Typography variant="caption" color="textSecondary">
+            <FormattedMessage defaultMessage="If selected customer will be able to choose this warehouse as pickup point. Ordered products can be shipped here from a different warehouse" />
+          </Typography>
+        </>
+      ),
+      value: WarehouseClickAndCollectOptionEnum.ALL
+    }
+  ];
+
+  const clickAndCollectChoicesPrivate = clickAndCollectChoicesPublic.filter(
+    choice => choice.value !== WarehouseClickAndCollectOptionEnum.LOCAL
+  );
+
+  return (
+    <Card>
+      <CardTitle
+        title={intl.formatMessage({
+          defaultMessage: "Settings",
+          description: "zones that warehouse sends to"
+        })}
+      />
+      <CardContent>
+        {renderCollection(
+          zones,
+          zone =>
+            maybe(
+              () => (
+                <div className={classes.link} key={zone.id}>
+                  <Link underline onClick={() => onShippingZoneClick(zone.id)}>
+                    {zone.name}
+                  </Link>
+                </div>
+              ),
+              <Skeleton />
+            ),
```

Comment:
```suggestion
            zone ? (
                <div className={classes.link} key={zone.id}>
                  <Link underline onClick={() => onShippingZoneClick(zone.id)}>
                    {zone.name}
                  </Link>
                </div>
              ) : <Skeleton />
```

Reply:
Done.  Whenever I've found `maybe` should I alter it to optional chaining operator, am I right?